src/main/java/com/epam/training/auction_backend/controllers/AuctionService.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.training.auction_backend.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.log4j.Logger;
+import org.hibernate.Query;
+import org.hibernate.Session;
+
+import com.epam.training.auction_backend.entity.Auction;
+
+public class AuctionService {
```

Comment:
Services shouldn't be in controllers package. Name should be changed because we don't need controllers in backend at all. Also, services should implement interface, especially those available remotely. Create interfaces in auctions_common and make services implement it.